code/script.js

```diff
@@ -1,19 +1,30 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 // Global variables, if you need any, declared here
+let questionNumber = 1
+let appointmentDate = '';
+let msgNotificationAudio = new Audio('./assets/notification.mp3');
+msgNotificationAudio.volume = 0.2;
+
```

Comment:
Great use of global variables and even more impressive with the notification sound.